### ISSUE TITLE
Confusion regarding AlexNet dense layer ordering

### ISSUE THREAD

**Issue by aksrajvanshi** [d2l-ai/d2l-en#1131]
@AnirudhDagar 

Problem: In the book, when AlexNet is implemented, in MxNet, it shows 

`nn.Dense(4096, activation="relu"), nn.Dropout(0.5)` which is understood to be in the order : 

dense layer -> relu -> dropout.

However, in pyTorch, the order seems to have changed and looks like this:

```
nn.Dropout(p=0.5),
nn.Linear(6400, 4096), nn.ReLU()
```

which seems like a different ordering than MxNet.

Can someone clarify this and edit the exact ordering accordingly. 

Thanks

**Comment by AnirudhDagar** [comment]
Thanks, @aksrajvanshi for pointing this out. This is indeed a mistake. Raised a PR to fix this.
